packages/webapp/src/containers/Task/TaskAssignment/index.jsx

```diff
@@ -1,14 +1,21 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import PureTaskAssignment from '../../../components/Task/PureTaskAssignment';
 import { loginSelector, userFarmEntitiesSelector, userFarmSelector } from '../../userFarmSlice';
 import { useDispatch, useSelector } from 'react-redux';
 import grabCurrencySymbol from '../../../util/grabCurrencySymbol';
-import { getCurrencyFromStore } from '../../../store/getFromReduxStore';
 import { HookFormPersistProvider } from '../../hooks/useHookFormPersist/HookFormPersistProvider';
-import { taskTypeSelector } from '../../taskTypeSlice';
 import { hookFormPersistSelector } from '../../hooks/useHookFormPersist/hookFormPersistSlice';
 import { createTask } from '../saga';
 import { useTranslation } from 'react-i18next';
+import { updateUserFarmWage, setUserFarmWageDoNotAskAgain } from '../../../containers/Task/saga';
```

Comment:
@SayakaOno 

This can be combined with `import { createTask } from '../saga';`